test_collections/matter/sdk_tests/support/performance_tests/utils.py

```diff
@@ -258,7 +258,6 @@ def generate_summary(
     summary_dict["test_summary_record"][
         "number_of_iterations_failed"
     ] = compute_count_state(execution_status, False)
-    summary_dict["test_summary_record"]["platform"] = "rpi"
```

Comment:
It seems this line was removed unintentionally

Reply:
Reverted changes to this file.